Build failure on macOS: `mmc_osx.c: error: ‘O_RDWR’ undeclared (first use in this function)`

```
/usr/bin/gcc-4.2 -std=gnu99 -DHAVE_CONFIG_H -I. -I..   -I/opt/local/include -Wall -D_FILE_OFFSET_BITS=64 -I./3rdparty/monocypher-3.1.3/src -I./3rdparty/monocypher-3.1.3/src/optional   -pthread  -pipe -Os -arch ppc -MT fwup-mmc_bsd.o -MD -MP -MF .deps/fwup-mmc_bsd.Tpo -c -o fwup-mmc_bsd.o `test -f 'mmc_bsd.c' || echo './'`mmc_bsd.c
mv -f .deps/fwup-mmc_bsd.Tpo .deps/fwup-mmc_bsd.Po
/usr/bin/gcc-4.2 -std=gnu99 -DHAVE_CONFIG_H -I. -I..   -I/opt/local/include -Wall -D_FILE_OFFSET_BITS=64 -I./3rdparty/monocypher-3.1.3/src -I./3rdparty/monocypher-3.1.3/src/optional   -pthread  -pipe -Os -arch ppc -MT fwup-mmc_linux.o -MD -MP -MF .deps/fwup-mmc_linux.Tpo -c -o fwup-mmc_linux.o `test -f 'mmc_linux.c' || echo './'`mmc_linux.c
mv -f .deps/fwup-mmc_linux.Tpo .deps/fwup-mmc_linux.Po
/usr/bin/gcc-4.2 -std=gnu99 -DHAVE_CONFIG_H -I. -I..   -I/opt/local/include -Wall -D_FILE_OFFSET_BITS=64 -I./3rdparty/monocypher-3.1.3/src -I./3rdparty/monocypher-3.1.3/src/optional   -pthread  -pipe -Os -arch ppc -MT fwup-mmc_osx.o -MD -MP -MF .deps/fwup-mmc_osx.Tpo -c -o fwup-mmc_osx.o `test -f 'mmc_osx.c' || echo './'`mmc_osx.c
mv -f .deps/fwup-fwup_xdelta3.Tpo .deps/fwup-fwup_xdelta3.Po
mmc_osx.c: In function ‘authopen_fd’:
mmc_osx.c:188: warning: implicit declaration of function ‘fork’
mmc_osx.c:191: warning: implicit declaration of function ‘open’
mmc_osx.c:191: error: ‘O_RDWR’ undeclared (first use in this function)
mmc_osx.c:191: error: (Each undeclared identifier is reported only once
mmc_osx.c:191: error: for each function it appears in.)
mmc_osx.c:195: warning: implicit declaration of function ‘close’
mmc_osx.c:195: error: ‘STDIN_FILENO’ undeclared (first use in this function)
mmc_osx.c:196: error: ‘STDOUT_FILENO’ undeclared (first use in this function)
mmc_osx.c:197: warning: implicit declaration of function ‘dup2’
mmc_osx.c:211: warning: implicit declaration of function ‘execvp’
/usr/bin/gcc-4.2 -std=gnu99 -DHAVE_CONFIG_H -I. -I..   -I/opt/local/include -Wall -D_FILE_OFFSET_BITS=64 -I./3rdparty/monocypher-3.1.3/src -I./3rdparty/monocypher-3.1.3/src/optional   -pthread  -pipe -Os -arch ppc -MT fwup-mmc_windows.o -MD -MP -MF .deps/fwup-mmc_windows.Tpo -c -o fwup-mmc_windows.o `test -f 'mmc_windows.c' || echo './'`mmc_windows.c
make[2]: *** [fwup-mmc_osx.o] Error 1
make[2]: *** Waiting for unfinished jobs....
```

--- a/src/mmc_osx.c
+++ b/src/mmc_osx.c
@@ -22,6 +22,8 @@
 #include <DiskArbitration/DiskArbitration.h>
 #include <IOKit/storage/IOStorageProtocolCharacteristics.h>
 
+#include <fcntl.h>
+#include <unistd.h>
 #include <sys/param.h>
 #include <sys/socket.h>
 #include <sys/stat.h>

